packages/components/src/components/brandshape/brandshape.ts

```diff
@@ -206,6 +260,60 @@ export default class SdBrandshape extends SolidElement {
       :host {
         @apply block;
       }
+
+      .container--stylized::before {
+        @apply absolute top-0 bottom-0 right-0 left-0 border-solid inset-0 z-[-1];
+        content: '';
+        background-color: var(--internal-color-background, white);
+        color: var(--internal-color, black);
+        border-color: var(--internal-border-color, black);
+        border-width: var(--internal-border-width, 0);
+        border-radius: 0 60px;
+        margin-bottom: calc(21.256% / 2);
+        margin-top: calc(21.256% / 2);
+        transform: skewY(-11deg);
+      }
+
+      .container--outline-primary::before {
+        --internal-color-background: none;
+        --internal-border-color: rgb(var(--sd-color-primary, 0 53 142));
+        --internal-border-width: 2px;
+      }
+
+      .container--outline-white::before {
+        --internal-color-background: none;
+        --internal-border-color: var(--sd-color-white, white);
+        --internal-border-width: 2px;
+      }
+
+      .image-wrapper {
+        @apply absolute top-0 left-0 w-full z-10 overflow-hidden;
+        height: -webkit-fill-available;
+        margin-bottom: calc(21.256% / 2);
+        margin-top: calc(21.256% / 2);
+        border-radius: 0 60px;
+        transform: skewY(-11deg);
+      }
+
+      slot[name='image']::slotted(img) {
+        @apply w-full h-full object-cover origin-top-left;
+        transform: skewY(11deg);
+      }
+
+      /* Responsive border-radius */
+      @media (min-width: 414px) {
+        .container--stylized::before,
+        .image-wrapper {
+          border-radius: 0 72px;
+        }
+      }
+
+      @media (min-width: 640px) {
+        .container--stylized::before,
+        .image-wrapper {
+          border-radius: 0 84px;
+        }
+      }
```

Comment:
@coraliefeil @EllaELSA  Should the border radius change on different screen sizes? (not component sizes)